docs/v1/P4Runtime-Spec.mdk

```diff
@@ -1306,7 +1316,8 @@ The `Action` message defines the following fields:
       on the way `Param` IDs should be allocated, as long as two `Param` of the
       same action do not have the same ID. Nonetheless, we recommend that the
       IDs be assigned incrementally, starting from 1, in the same order as in
-      the P4 action declaration.
+      the P4 action declaration.  The programmer can chose the IDs using the
```

Comment:
minor typo: chose -> choose

Reply:
and similar in the next "hunk" of diff below, and perhaps others I have not read yet.

Reply:
Done.